packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -186,7 +186,7 @@ export const getSyncs = async (
                             )
                         END as latest_sync `),
                     db.knex.raw(
-                        `ROW_NUMBER() OVER (PARTITION BY ${SYNC_JOB_TABLE}.sync_id ORDER BY ${SYNC_JOB_TABLE}.deleted ASC, ${SYNC_JOB_TABLE}.updated_at DESC) as job_row_number`
+                        `ROW_NUMBER() OVER (PARTITION BY ${TABLE}.id, ${SYNC_JOB_TABLE}.sync_id ORDER BY ${SYNC_JOB_TABLE}.updated_at DESC) as job_row_number`
```

Comment:
also deleting the `ORDER BY ${SYNC_JOB_TABLE}.deleted ASC` because we always filter on `${SYNC_JOB_TABLE}.deleted` [below](https://github.com/NangoHQ/nango/pull/2437/files#diff-7be9b1d2fee1ed263243cfe652dbb029abdc165631a55b0677119bf8fb3a743cR205)